pageserver/src/http/openapi_spec.yml

```diff
@@ -82,11 +82,6 @@ paths:
         schema:
           type: string
           description: Controls calculation of current_logical_size_non_incremental
-      - name: include-non-incremental-physical-size
```

Comment:
Is this used from the console? I guess not, or the end-to-end tests would fail. Todo: remove it from the cloud repo's autogenerated files.

Reply:
https://github.com/neondatabase/cloud/pull/3358